File: src/misc/Julia-functions/split_str2int.jl
"""
    split_str2int(input::String, header::Bool=true, splitter::Char=' ')

Splits an `input` string into Int64 array removing `header` (if true) and splitting at `splitter` character.
"""

function split_str2int(input::String, header::Bool=true, splitter::Char=' ')
    s = split(input, splitter)
    if header
        s = s[2:end]
    end
    a = zeros(length(s))
    for n in 1:length(s)
        a[n] = parse(Int64, s[n])
    end
    a = Array{Int64}(a)
    return a
end

File: src/misc/MATLAB-functions/split_str2int.m
function result = split_str2int(input, header, splitter)
    % SPLIT_STR2INT(input, header, splitter)
    % Splits an input string into Int64 array removing header (if true) and splitting at splitter character.

    if isempty(header)
        header = true;
    end
    if isempty(splitter)
        splitter = ' ';
    end
    s = split(input, splitter);
    s = string(s);
    if header
        s = s(2:end);
    end
    result = zeros(length(s), 1);
    for n = 1:length(s)
        result(n) = str2num(s(n))
    end
end